.github/workflows/test-template.yml

```diff
@@ -121,7 +121,7 @@ jobs:
         shard: ['1/5', '2/5', '3/5', '4/5', '5/5']
         queryEngine: ${{ fromJson(needs.determineBuildMatrixParams.outputs.queryEngine) }}
         engineProtocol: ['graphql', 'json']
-        node: [16, 18]
+        node: [16, 18, 20]
```

Comment:
We might not need 16 here if we are testing it builtkite. At least that was the reason why 14 is not there AFAIK.

Reply:
yes that's what we used to do indeed, I'll remove 16 here